<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QExplorerFinderClass</class>
 <widget class="QWidget" name="QExplorerFinderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Explorer Finder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpFolder">
     <property name="title">
      <string>Debug</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditFolder">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="grpSearch">
       <property name="title">
        <string>Search</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="lineEditSearch">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSearch">
          <property name="text">
           <string>Search</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listResults"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutSelectionControls">
     <item>
      <widget class="QLabel" name="lblStatus">
       <property name="text">
        <string>Found: 0 | Selected: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chkDefaultSelectAll">
       <property name="text">
        <string>Default Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnListSelectAll">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnListInvert">
       <property name="text">
        <string>Invert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnListNone">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutActions">
     <item>
      <widget class="QCheckBox" name="chkCloseAfterSelect">
       <property name="text">
        <string>Close after selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkAlwaysOnTop">
       <property name="text">
        <string>Always on top</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectAll">
       <property name="text">
        <string>Select in Explorer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QExplorerFinder.qrc"/>
 </resources>
 <connections/>
</ui>
